Santillana-Demo-AR-Experiencia-Residuos/ModelNode.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  ModelNode.swift
+//  Santillana-Demo-AR-Experiencia-Residuos
+//
+//  Created by Carlos Daniel Hernandez Chauteco on 6/20/19.
+//  Copyright © 2019 Alejandro Mendoza. All rights reserved.
+//
+import SceneKit
+
+class ModelNode{
+    static let share = ModelNode()
+    
+    func lata() -> SCNNode {
+        guard let pathScene = Bundle.main.path(forResource: "lataX", ofType: "scn", inDirectory: "art.scnassets/Models") else { return SCNNode() }
```

Comment:
All functions in this class are repetitive, don't you think it would be better to create a general function that accepts a model name and retrieve a SCNode?